### ISSUE TITLE
svnadmin: Can't open file '/etc/subversion/servers': Permission denied

### ISSUE THREAD

**Issue by sdt** [sdt/Test-SVN-Repo#1]
http://www.cpantesters.org/cpan/report/f02cff56-cb25-11e0-a2c2-a878e7bc264d

Can repro this with chmod 600 /etc/subversion/servers

Lots of these:

```
#   Failed test '... ctor lives'
#   at t/multiple.t line 29.
# died: svnadmin: Can't open file '/etc/subversion/servers': Permission denied
#  at t/multiple.t line 29
```

Need to just BAIL_OUT in this case I think. Can't find a simple case like can_run('svnadmin') though

**Comment by sdt** [comment]
http://www.cpantesters.org/cpan/report/d9c697b0-cbeb-11e0-8784-9880e7bc264d

(same tester, new package)

**Comment by sdt** [comment]
Addressed in this commit - slated for 0.007 release

https://github.com/sdt/Test-SVN-Repo/commit/40c920b97e1f51f22be4a9029b0aa449e4c2b08f
